### PR TITLE
reverting to prebrax

### PR DESCRIPTION
**Pull request by nathanjzhao**
going forward will try to get ppo first then vmap/pmap, in spite of long iteration times 😔 .

a few tests are here. i think if this is not successful (im going to try a few things) im just going to try to unroll what brax does in psuedocode then reimplement with less abstractions

### CODE REVIEW

File: train.py [hunk 1]
```diff
@@ -16,23 +16,23 @@
 
 @dataclass
 class Config:
-    lr_actor: float = field(default=0.0003)
-    lr_critic: float = field(default=0.0003)
+    lr_actor: float = field(default=3e-6)
+    lr_critic: float = field(default=3e-6)
```
Comment (codekansas) [inline train.py]: `3e-4` is the golden learning rate :)

**Comment by codekansas** [comment]
nice :)

File: train.py (hunk 1 above)
Comment (nathanjzhao) [reply train.py]: yea thats what i was thinking -- just messing around a bit

**Comment by nathanjzhao** [comment]
The training loop does work now -- at least, the score increases over time (given the correct hyperparams. lambda/gamma seems to be different in diff/codebases literature so messing around with that. Also just saw your comment so I may have wasted a little time doing other learning rates...). However, score increasing over time seems to be a good signifier.

Problem is that I can't just imitate brax's rewards exactly, because they have an is_healthy variable that becomes 0 when the humanoid is no longer healthy (I think the other environments that are still "alive" keep going on for a while so that the effect of the 0 is more felt). So, there's no real punishment if the robot just dies, because the environment auto-restarts it re-accumulates reward. So, my reward here is a bit different, just optimizing height. This is a bit of a problem though because the bot likes being as straight up as possible which isn't really good for actually being stable in standing.

Not sure what next steps are would be, unless just making separate environments (vmapping) is the solution.

Also I think I'll stick with torch as long as possible (maybe with @torch.jit.export) until need to parallelize. Maybe I'll use fairscale :^)
